src/main/java/vct/main/Main.scala

```diff
@@ -97,7 +97,8 @@ class Main {
     clops.add(learn.getEnable("Learn unit times for AST nodes."), "learn")
     CommandLineTesting.addOptions(clops)
     Configuration.add_options(clops)
-    clops.parse(args)
+    val VeyMontFiles = Array("examples/veymont-src/channel.pvl", "examples/veymont-src/barrier.pvl")
```

Comment:
why is this hardcoded?

Reply:
Because it is fixed input for every run of VeyMont. Is there a better/nicer way to include this code?

Reply:
In that case it is better to make it a resource, as are `cuda.h` and such. They live in `vercors/src/main/universal/res/include`, can you put them there? I would then make a method similar to `Configuration.getCIncludePath` for the two files. (I would prefer to also start splitting included stuff out by language, but we can do that later)